src/components/Profile/components/Identification/index.tsx

```diff
@@ -26,6 +29,7 @@ import Cropper, { ReactCropperElement } from 'react-cropper';
 import ZoomEvent from 'react-cropper';
```

Comment:
@abelyshev22 Can you verify that it's safe to remove this line, and remove it to fix the warning?  (Probably just remove it and then test that it works and the warning goes away.)

Reply:
(You didn't cause the problem, but since you're working in the file it's a good opportunity to clean it up.)